src/nni_manager/training_service/local/localTrainingService.ts

```diff
@@ -295,7 +297,9 @@ class LocalTrainingService implements TrainingService {
 
     public cleanUp(): Promise<void> {
         this.stopping = true;
-
+        for(const stream of this.streams) {
+            stream.destroy();
+        }
```

Comment:
Is it proper to close all the opened streams at the very end?

Reply:
I think that's OK by now, will find a better way in future.